monitor/src/auth/auth.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { JwtService } from '@nestjs/jwt';
+import { PASSWORD } from 'src/common/configuration';
+
+@Injectable()
+export class AuthService {
+  constructor(
+    private jwtService: JwtService
+  ) {}
+
+  async signIn(pass) {
+    console.log(PASSWORD)
```

Comment:
@joseph-bisonai It would be better if we do not log any password.
```suggestion
```

Reply:
my bad. I should have removed this